gradle/enforcement/codenarc.groovy

```diff
@@ -308,7 +308,8 @@ ruleset {
   FieldName {
     regex = '^_?[a-z][a-zA-Z0-9]*$'
     finalRegex = '^_?[a-z][a-zA-Z0-9]*$'
-    staticFinalRegex = '^logger$|^[A-Z][A-Z_0-9]*$|^serialVersionUID$'
+    // can be either constant (ABC_XYZ) or non-constant (abcXyz)
+    staticFinalRegex = '^[A-Z][A-Z_0-9]*$|^_?[a-z][a-zA-Z0-9]*$'
```

Comment:
Isn't it equivalent to just `^_?[a-z][a-zA-Z0-9]*$`?

Reply:
Hmm, this one won't fail something like `_ABC` - I don't know if that's an issue though

Reply:
ya, i was being lazy and just copy-pasted and or-ed them and didn't apply any thinking 😄, i don't think we use leading underscores anywhere